glide-core/src/protobuf/redis_request.proto

```diff
@@ -107,6 +107,7 @@ enum RequestType {
     HLen = 69;
     Echo = 70;
     ZPopMin = 71;
+    Lindex = 73;
```

Comment:
change from 73 to 72

Reply:
Strlen is already 72, just didnt been merged yet :)

Reply:
Since we don't know what would be merged first - lets keep the current PR uptodate with the current commits